internal/controllers/clusterextension_admission_test.go

```diff
@@ -402,6 +388,66 @@ func TestClusterExtensionAdmissionServiceAccount(t *testing.T) {
 	}
 }
 
+func TestClusterExtensionAdmissionInstall(t *testing.T) {
+	oneOfErrMsg := "at least one of [preflight] are required when install is specified"
+
+	testCases := []struct {
+		name          string
+		installConfig *ocv1alpha1.ClusterExtensionInstallConfig
+		errMsg        string
+	}{
+		{
+			"install specified, nothing configured",
```

Comment:
Add field names to the struct instances?

Reply:
https://github.com/operator-framework/operator-controller/pull/1439/commits/1685475e0d8d5aac66e7abf7388f31b09bc7525c